CHANGELOG.md

```diff
@@ -45,6 +45,9 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed an issue where a message about changed metadata would occur on saving although nothing changed. [#9159](https://github.com/JabRef/jabref/issues/9159)
 - When adding or editing a subgroup it is placed w.r.t. to alphabetical ordering rather than at the end. [koppor#577](https://github.com/koppor/jabref/issues/577)
 - We modified the Directory of Open Access Books (DOAB) fetcher so that it will now also fetch the ISBN when possible. [#8708](https://github.com/JabRef/jabref/issues/8708)
+- We modified the "Delete file" dialog and add the full file path to the dialog text. The file path in the title was changed to file name only. [koppor#534](https://github.com/koppor/jabref/issues/534)
+- Download from URL now automatically fills with URL from clipboard. [koppor#535](https://github.com/koppor/jabref/issues/535)
+- We added HTML and Markdown files to Find Unlinked Files and removed BibTex. Markdown was added as a StandardFileType and StandardExternalFileType [koppor#547](https://github.com/koppor/jabref/issues/547)
```

Comment:
```suggestion
- We added HTML and Markdown files to Find Unlinked Files and removed BibTex. [koppor#547](https://github.com/koppor/jabref/issues/547)
```
The changelog is targeted to end users, so implementation details are not necessary here